eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -776,24 +779,35 @@ object Router {
     */
   val ROUTE_MAX_LENGTH = 20
 
+  // The default amount of routes we'll search for when findRoute is called
+  val DEFAULT_ROUTES_AMOUNT = 3
```

Comment:
nitpick: "amount" is misleading
```suggestion
  val DEFAULT_ROUTES_COUNT = 3
```

Reply:
Done